src/platforms/rp2040/src/lib/CMakeLists.txt

```diff
@@ -44,6 +46,10 @@ set(
     ${CMAKE_SYSTEM_NAME}-${CMAKE_SYSTEM_PROCESSOR}
 )
 
+add_compile_definitions(OS_VERSION_MAJOR=${PICO_SDK_VERSION_MAJOR})
+add_compile_definitions(OS_VERSION_MINOR=${PICO_SDK_VERSION_MINOR})
+add_compile_definitions(OS_VERSION_PATCH=${PICO_SDK_VERSION_REVISION})
+
```

Comment:
I couldn't find anything better, it looks like the usual approach.

Also, we may want an additional avm_ or atomvm_* key for the chip versions. Pico SDK has functions to query chip and rom versions. This could be in another PR.

Reply:
This would be good. Perhaps at the same time the ESP32 key `esp_chip_info_atom` should be renamed, so that both (and eventually other) platforms can use the same key?

Reply:
> Also, we may want an additional avm_ or atomvm_* key for the chip versions. Pico SDK has functions to query chip and rom versions. This could be in another PR.

This would be good. Perhaps at the same time the ESP32 key esp_chip_info_atom should be renamed, so that both (and eventually other) platforms can use the same key?

Reply:
But maybe that kind of breakage needs to be based on main (0.7.0)?